nixos/tests/networking/networkd-and-scripted.nix

```diff
@@ -819,34 +690,33 @@ let
           '';
         };
       };
-      nodes.client_with_privacy = { pkgs, ... }: with pkgs.lib; {
+      nodes.client_with_privacy = {
```

Comment:
something that i tend to stylistically do is always make nixos modules functions, since there are iirc some nasty edge cases with making them attrsets, but i don't remember exactly what the edge cases were so....... again you can totally ignore this, i am just writing what i am thinking.

Reply:
I'll ignore that since other tests do it as well and imo it is cleaner without having it.